libkbfs/block_journal_test.go

```diff
@@ -708,7 +708,7 @@ func TestBlockJournalFlushMDRevMarker(t *testing.T) {
 		bcache, reporter, tlfID, CanonicalTlfName("fake TLF"),
 		entries)
 	require.NoError(t, err)
-	err = j.removeFlushedEntries(ctx, entries, tlfID, reporter)
+	_, err = j.removeFlushedEntries(ctx, entries, tlfID, reporter)
```

Comment:
I think a previous version of this test checked the returned `unflushedBytes`. I'll have to dig it out of the history.